core/kazoo_number_manager/src/knm.hrl

```diff
@@ -2,18 +2,19 @@
 -include_lib("kazoo_stdlib/include/kz_databases.hrl").
 -include("knm_phone_number.hrl").
 
--define(APP, kazoo_number_manager).
+-ifdef(TEST).
+-include_lib("eunit/include/eunit.hrl").
```

Comment:
Every time this file is included compilation times are way worse.
Streaming to group `user` is very fine, compared to including all of this just for `?debugFmt`.